tests/unit/components/pacbio/PacbioWellModal.spec.js

```diff
@@ -67,8 +69,11 @@ describe('PacbioRunInfo', () => {
     it('has a table of well libraries', () => {
       expect(wrapper.find('#wellLibraries')).toBeDefined()
     })
+    it('has a Extension Time input', () => {
+      expect(wrapper.find('.insertSize')).toBeDefined()
+    })
```

Comment:
`Extension Time input` doesn't seem to match `.insertSize` - is this a copy-paste error?

Reply:
yep must be a test i missed false passing